types/signed_msg_type.go

```diff
@@ -1,11 +1,20 @@
 package types
 
-import cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
+import cmtproto "github.com/cometbft/cometbft/api/cometbft/types/v1"
+
+type SignedMsgType = cmtproto.SignedMsgType
+
+const (
+	SignedMsgType_UNKNOWN   SignedMsgType = cmtproto.UnknownType
+	SignedMsgType_PREVOTE   SignedMsgType = cmtproto.PrevoteType
+	SignedMsgType_PRECOMMIT SignedMsgType = cmtproto.PrecommitType
+	SignedMsgType_PROPOSAL  SignedMsgType = cmtproto.ProposalType
+)
```

Comment:
This is some activism on my part. Those const names differ from the naming convention used by gogoproto for other enums by default, and they are also poorly namespaced: it's not clear at a glance what "UnknownType" refers to, among the dozens of names that are defined in `types`. So I brought them in line with the general convention, even though the proto source gives the name overrides specifically for Go. I can revert these to the previously established short names, since everybody should have gotten used to them by now.

Reply:
Reverted in https://github.com/cometbft/cometbft/pull/495/commits/57e8a818633132ac475f5a7bbb8302b3b0b9766f